gov2/cloudwatch/README.md

```diff
@@ -1,108 +1,54 @@
+- As an AWS best practice, grant this code least privilege, or only the 
+  permissions required to perform a task. For more information, see 
+  [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege) 
+  in the *AWS Identity and Access Management 
+  User Guide*.
+- This code has not been tested in all AWS Regions. Some AWS services are 
+  available only in specific Regions. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/) on the AWS website.
+- Running this code might result in charges to your AWS account.
 
-### GetMetricData/GetMetricDatav2.go
+## Code examples
 
-This example displays the metric data points for the provided input in the given time-frame.
+### Single actions
 
-`go run CreateCustomMetricv2.go -mN METRIC-NAME -n NAMESPACE -dn DIMENSION-NAME -dv DIMENSION-VALUE -id ID -dM DIFFINMINUTES -s STAT -p PERIOD`
+- [Create a custom metric](CreateCustomMetric/) (`CreateCustomMetric`)
+- [Create an alarm that watches a metric](CreateEnableMetricAlarm/) (`CreateEnableMetricAlarm`)
+- [Describe alarms for a metric](DescribeAlarms/) (`DescribeAlarms`)
+- [Disable alarm actions](DisableAlarm/) (`DisableAlarm`)
+- [Get logged events](GetLogEvents/) (`GetLogEvents`)
+- [Get metric statistics](GetMetricData/) (`GetMetricData`)
+- [List metrics](ListMetrics/) (`ListMetrics`)
+- [Put an event](PutEvent/) (`PutEvent`)
 
-- _NAMESPACE_ is the namespace for the metric.
-- _METRIC-NAME_ is the name of the metric.
-- _DIMENSION-NAME_ is the name of the dimension.
-- _DIMENSION-VALUE_ is the value of the dimension.
-- _ID_ is a short name used to tie the object to the results in the response
-- _DIFFINMINUTES_ is the difference in minutes for which the metrics are requested
-- _STAT_ is the Statistic to return i.e. SUM, COUNT, AVERAGE etc
-- _PERIOD_ is the granularity, in seconds, of the returned data points
+### Run the examples
 
-The unit test accepts similar values in _config.json_
+Go to the directory where you want to run the sample, and do the following:
 
-### Notes
+```
+go mod tidy
+go run .
+```
+### Prerequisites
 
-- We recommend that you grant this code least privilege,
-  or at most the minimum permissions required to perform the task.
-  For more information, see
-  [Grant Least Privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
-  in the AWS Identity and Access Management User Guide.
-- This code has not been tested in all AWS Regions.
-  Some AWS services are available only in specific
-  [Regions](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
-- Running this code might result in charges to your AWS account.
+Prerequisites for running the examples for this service can be found in the
+[README](../README.md#Prerequisites) in the GoV2 folder.
 
-## Running the unit tests
+### Tests
 
```

Comment:
Missing notice about "test might incur charges" from the template.

Reply:
Notice added